mlir-sycl/lib/Dialect/SYCL/IR/SYCLHostTraits.cpp

```diff
@@ -0,0 +1,19 @@
+//===--- SYCLHostTraits.cpp -----------------------------------------------===//
+//
+// MLIR-SYCL is under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "mlir/Dialect/SYCL/IR/SYCLHostTraits.h"
+
+#include "mlir/Dialect/LLVMIR/LLVMTypes.h"
+
+using namespace mlir;
+using namespace mlir::sycl;
+
+LogicalResult mlir::sycl::verifySYCLHandlerOpTrait(Operation *op) {
```

Comment:
[nit]: Given this file has the using declaration at lines 13-14 the prefix "mlir::sycl::" is not strictly necessary.

Reply:
I did this because of [that's what the LLVM Coding Standards](https://llvm.org/docs/CodingStandards.html#use-namespace-qualifiers-to-implement-previously-declared-functions) recommend